tests/etl/test_runnable_jobs.py

```diff
@@ -38,63 +71,22 @@ def test_prune_old_runnable_job(test_repository, eleven_jobs_stored):
 
 @responses.activate
 def test_taskcluster_runnable_jobs(test_repository):
-
     """
     Test getting runnable jobs without providing decision task id
     """
     repo = test_repository.name
-    task_id = 'AFq3FRt4TyiTwIN7fUqOQg'
-    tc_index_url = TASKCLUSTER_INDEX_URL % repo
-    tc_graph_url = TASKCLUSTER_TASKGRAPH_URL.format(task_id=task_id)
-    platform = 'plaform name'
-    grp_symbol = 'GRP'
-    grp_name = 'Group Name'
-    symbol = 'sym'
-    collection = {'opt': True}
-    name = 'job name'
-    description = 'job description'
-
-    content1 = {
-        'taskId': task_id
-    }
-
-    content2 = {
-        'node': {
-            'task': {
-                'extra': {
-                    'treeherder': {
-                        'groupSymbol': grp_symbol,
-                        'groupName': grp_name,
-                        'symbol': symbol,
-                        'machine': {
-                            'platform': platform
-                        },
-                        'collection':  collection
-                    }
-                },
-                'metadata': {
-                    'name': name,
-                    'description': description
-                }
-            }
-        }
-    }
 
-    responses.add(responses.GET, tc_index_url, json=content1, match_querystring=True, status=200)
-    responses.add(responses.GET, tc_graph_url, json=content2, match_querystring=True, status=200)
+    compressed = StringIO.StringIO()
+    with gzip.GzipFile(fileobj=compressed, mode="w") as f:
```

Comment:
With the below switched to use `fetch_json()` and requests' automatic gzip handling, the tests should be fine using `json=`